src/core/xref.js

```diff
@@ -47,9 +54,15 @@ function createXrefMap(elems) {
  * @returns {Object} { keys: [{ term }] }
  */
 function createXrefQuery(xrefs) {
-  const queryKeys = [...xrefs.keys()].reduce((queryKeys, term) => {
-    return queryKeys.add(JSON.stringify({ term }));
-  }, new Set());
+  const queryKeys = [...xrefs.entries()].reduce(
+    (queryKeys, [term, entries]) => {
+      entries.forEach(({ specs }) => {
```

Comment:
nit: maybe do `for of ` in here... having `.forEach` inside something you are already seems out of place.

Reply:
Fixed typo above.